mirdata/datasets/dagstuhl_choirset.py

```diff
@@ -0,0 +1,465 @@
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_spr_path)
+
+    def to_jams(self):
+        """Jams: the track's data in jams format"""
+
+        beat_data = [(self.beat, "beat")] if self.beat else None
+
+        return jams_utils.jams_converter(
+            audio_path=self.audio_stm_path, beat_data=beat_data
+        )
+
+
+@io.coerce_to_bytes_io
+def load_audio(fhandle: BinaryIO) -> Tuple[np.ndarray, float]:
+    """Load a Dagstuhl ChoirSet audio file.
+
+    Args:
+        audio_path (str): path pointing to an audio file
+
+    Returns:
+        * np.ndarray - the audio signal
+        * float - The sample rate of the audio file
+
+    """
+    return librosa.load(fhandle, sr=22050, mono=True)
+
+
+@io.coerce_to_string_io
+def load_f0(fhandle: TextIO) -> annotations.F0Data:
+    """Load a Dagstuhl ChoirSet F0-trajectory.
+
+    Args:
+        fhandle (str or file-like): File-like object or path to F0 file
+
+    Returns:
+        F0Data Object - the F0-trajectory
+    """
+    times = []
+    freqs = []
+    confs = []
+    reader = csv.reader(fhandle, delimiter=",")
+    for line in reader:
+        times.append(float(line[0]))
+        freqs.append(float(line[1]))
+        if len(line) == 3:
+            confs.append(float(line[2]))
+        else:
+            confs.append(float(1.0))
+
+    times = np.array(times)
+    freqs = np.array(freqs)
+    confs = np.array(confs)
+    return annotations.F0Data(times, freqs, confs)
+
+
+@io.coerce_to_string_io
+def load_score(fhandle: TextIO) -> annotations.NoteData:
+    """Load a Dagstuhl ChoirSet time-aligned score representation.
+
+    Args:
+        fhandle (str or file-like): File-like object or path to score representation file
+
+    Returns:
+        NoteData Object - the time-aligned score representation
+    """
+    intervals = np.empty((0, 2))
+    notes = []
+    reader = csv.reader(fhandle, delimiter=",")
+    for line in reader:
+        intervals = np.vstack([intervals, [float(line[0]), float(line[1])]])
+        notes.append(float(line[2]))
+
+    return annotations.NoteData(intervals, librosa.midi_to_hz(notes), None)
+
+
+@io.coerce_to_string_io
+def load_beat(fhandle: TextIO) -> annotations.BeatData:
+    """Load a Dagstuhl ChoirSet beat annotation.
+
+    Args:
+        fhandle (str or file-like): File-like object or path to beat annotation file
+
+    Returns:
+        BeatData Object - the beat annotation
+    """
+    times = []
+    positions = []
+    position = 0
+    reader = csv.reader(fhandle, delimiter=",")
+    for line in reader:
+        times.append(float(line[0]))
+        raw_position = float(line[1])
+        if np.floor(raw_position) == raw_position:
+            position = 1
+        else:
+            position += 1
+        positions.append(position)
+
+    times = np.array(times)
+    positions = np.array(positions)
+    return annotations.BeatData(times, positions)
```

Comment:
to fix the mypy error, replace these 3 lines with:
`return annotations.BeatData(np.array(times), np.array(positions))`